src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -41,6 +41,7 @@ const CreateConceptForm = (props) => {
         && map.map_type !== MAP_TYPE.conceptSet,
   );
   const descriptions = props.existingConcept.descriptions || props.description;
+  const showCodedOption = concept === 'Question' || concept === '' || isEditConcept;
```

Comment:
Good work on this @tejiri4.
Could you make use of the `CONCEPT_CLASS.question` constant here?